Adding documentation for new heartbeat accessors (#2250)
timescale/docs: api/_hyperfunctions/heartbeat_agg/num_live_ranges.md

---
api_name: num_live_ranges()
excerpt: Count the number of live ranges
topics: [hyperfunctions]
api:
  license: community
  type: function
  toolkit: true
  version:
    stable: 1.16.0
hyperfunction:
  family: state tracking
  type: accessor
  aggregates:
    - heartbeat_agg()
api_details:
  summary: |

>     Given a heartbeat aggregate, this will return the number of live periods.

Reviewer: ```suggestion
    Given a heartbeat aggregate, this returns the number of live periods.
```

  signatures:
    - language: sql
      code: |
        num_live_ranges(
            agg HEARTBEATAGG
        ) RETURNS BIGINT
  parameters:
    required:
      - name: agg
        type: HeartbeatAgg
        description: >
          A heartbeat aggregate to get the number of ranges from.
    returns:
      - column: num_live_ranges
        type: bigint
        description: >
          The number of live ranges in the aggregate.
  examples:
    - description: >

>         Given a table called `liveness` containing weekly heartbeat aggregates in column `health` with timestamp column `date`, we can use the following to see how many intervals the system was up in a given week.

Reviewer: ```suggestion
        Given a table called `liveness` containing weekly heartbeat aggregates in column `health` with timestamp column `date`, use this query to see how many intervals the system was up in a given week:
```
